### PR TITLE
Fixes bombs breaking

### LINKED ISSUE #22341: You can disarm syndiebombs by bashing them until they break
How to reproduce
- Find a syndiebomb, armed or not
- Bash it with something for a little bit (or blow it up)
- It vanishes and drops a bomb core

### PR DESCRIPTION
**Pull request by Cyberboss**
PLS TAG AS "FUCK" AND "I DED"

FUCKING FIXES #22341

### CODE REVIEW

**Comment by jughu** [comment]
Make it 10% to explode everytime you bash it

**Comment by KorPhaeron** [comment]
Doesn't this mean you can bypass the whole timing/warning sequence by just hitting it until it explodes?

**Comment by jughu** [comment]
I mean you can technically already do this by using minibombs

**Comment by Cyberboss** [comment]
@KorPhaeron I'd argue that if you really want a bomb to go off, it's easy enough to cut all the wires

File: code/game/machinery/syndicatebomb.dm [hunk 1]
```diff
@@ -166,7 +171,10 @@
 			new /obj/item/stack/sheet/plasteel( loc, 5)
 			qdel(src)
 	else
+		var/old_integ = obj_integrity
 		. = ..()
+		if((old_integ < obj_integrity) && active && !defused && (payload in src))
```
Comment (Cyberboss) [inline code/game/machinery/syndicatebomb.dm]: merp

**Comment by Firecage** [comment]
Cyber, I would only agree with this if the Training bombs wires is the same as the wires of real bombs.

**Comment by optimumtact** [comment]
8 commits

**Pull request merged**